guides/common/modules/proc_creating-image-only-hosts.adoc

```diff
@@ -23,6 +24,8 @@ ifdef::azure-provisioning[* The *Azure Subnet* field is populated with the requi
 ifdef::azure-provisioning[. Optional: If you want to use a static private IP address, from the *IPv4 Subnet* list select the {Project} subnet with the *Network Address* field matching the Azure subnet address. In the *IPv4 Address* field, enter an IPv4 address within the range of your Azure subnet.]
 . Click the *Operating System* tab, and confirm that all fields automatically contain values.
 ifdef::openstack-provisioning[. If you want to change the image that populates automatically from your compute profile, from the *Images* list, select a different image to base the new host's root volume on.]
+ifdef::azure-provisioning[. For *Provisioning Method*, ensure *Image Based* is selected.]
+ifdef::azure-provisioning[. In the *Image* field, you can select the Azure Resource Manager image, including any custom or shared image gallery image created.]
```

Comment:
You cannot select in the field. You can either enter an image in the field or select an image from the list. Also, I think it is redundant to specify what images can users choose here. Just saying "the Azure Resource Manager image that you want to use for provisioning" would do the trick.

Reply:
Fixed, thanks.